tracker_linux.go

```diff
@@ -140,13 +140,14 @@ func (t *linuxProcessTracker) doCleanupByEnvironmentVariable() error {
 // cleanupProcess terminates the process given by its PID. If the process has
 // already terminated, this will not return an error.
 func cleanupProcess(pid int) error {
-	catcher := grip.NewBasicCatcher()
 	// A process returns syscall.ESRCH if it already terminated.
-	if err := syscall.Kill(pid, syscall.SIGTERM); err != syscall.ESRCH {
-		catcher.Add(errors.Wrapf(err, "failed to send sigterm to process with pid '%d'", pid))
-		catcher.Add(errors.Wrapf(syscall.Kill(pid, syscall.SIGKILL), "failed to send sigkill to process with pid '%d'", pid))
+	if err := syscall.Kill(pid, syscall.SIGTERM); err != nil && err != syscall.ESRCH {
```

Comment:
if sending a SIGTERM errors why would we be able to send a SIGKILL?

Reply:
SIGTERM can be ignored, SIGKILL can't.

Reply:
I was assuming syscall.Kill will return an error only if there's a problem _sending the signal_, in which case we wouldn't be able to send SIGKILL either. 
In my mental model syscall.Kill just sends the signal doesn't wait to find out what the process does with it.

Reply:
True. It's easier to just send both SIGTERM and SIGKILL and ignore ESRCH in both cases, so I just changed it to ignore ESRCH for SIGKILL.